server/svix-server/tests/utils/mod.rs

```diff
@@ -223,7 +223,12 @@ pub fn get_default_test_config() -> ConfigurationInner {
     let _ = dotenv::dotenv();
     let cfg = svix_server::cfg::load().unwrap();
 
-    cfg.as_ref().clone()
+    let mut cfg = cfg.as_ref().clone();
+
+    // Ensure operational webhooks are turned off for tests by default
+    cfg.operational_webhook_address = None;
```

Comment:
Why would you need to be explicit about it? Just leave it how it was, it'll use the default value.